tests/std/tests/LWG2381_num_get_floating_point/test.cpp

```diff
@@ -532,6 +532,23 @@ void test_gh_3378() {
     }
 }
 
+// Also test GH-3980 <iostream>: Wrong reading of float values
+template <class Flt>

```

Comment:
Not worth resetting testing: vwls pls.
`Flt` isn't _that_ much shorter than `Float`. If you want a descriptive name, be descriptive. If you want a short name, `T` would work fine here and honestly `std::floating_point T` (or `static_assert(std::is_floating_point_v<T>)` in the function body) would be even better.

Reply:
`Float` specifically is dangerous due to the potential confusion with `float`. `Floating` or any other name avoids that hazard.